.github/workflows/auto_cherry_pick.yml

```diff
@@ -56,6 +56,7 @@ jobs:
           labels: |
             Auto_Cherry_Picked
             ${{ matrix.label }}
+            No-CherryPick
```

Comment:
Instead of copying the label No-CherryPick, we could restrict the https://github.com/SatelliteQE/robottelo/blob/d7078f2aada41a8a583d2b37eb0021b24a9af7cc/.github/workflows/required_labels.yml#L7 for default branch(master). This will avoid even the adding label and no need to run the workflow as well

Reply:
i like that

Reply:
We could not and should not because we are not always going to cherrypick just from master. We might remove something from 6.13 and in subsequent zStream branches but not in master.

So to support this scenario we have to maintain the ACP GHA in all branches. 

Its better to have some support than no support. :)